addons/website_sale/views/templates.xml

```diff
@@ -1148,24 +1148,43 @@
-                                            <div class="container">
-                                                <a t-att-href="website.contact_us_button_url"
-                                                   class="btn btn-primary btn_cta">Contact Us
-                                                </a>
-                                            </div>
-                                        </section>
-                                    </div>
-                                    <t t-if="is_view_active('website_sale.product_tags')" t-call="website_sale.product_tags">
-                                        <t t-set="all_product_tags" t-value="product_variant.all_product_tag_ids"/>
-                                    </t>
                                 </div>
                             </form>
+                            <t t-call="website_sale.product_accordion"/>
+                            <div
+                                id="contact_us_wrapper"
+                                t-attf-class="{{'d-flex' if combination_info['prevent_zero_price_sale'] else 'd-none'}} oe_structure oe_structure_solo #{_div_classes}"
+                            >
+                                <section
+                                    class="s_text_block"
+                                    data-snippet="s_text_block"
+                                    data-name="Text"
+                                >
+                                    <div class="container">
+                                        <a
+                                            t-att-href="website.contact_us_button_url"
+                                            class="btn btn-primary btn_cta"
+                                        >
+                                            Contact Us
+                                        </a>
+                                    </div>
+                                </section>
+                            </div>
+                            <t
+                                t-if="is_view_active('website_sale.product_tags')"
+                                t-call="website_sale.product_tags"
+                            >
+                                <t
+                                    t-set="all_product_tags"
+                                    t-value="product_variant.all_product_tag_ids"
+                                />
+                            </t>
                             <p t-elif="not product.active" class="alert alert-warning">This product is no longer available.</p>
                             <p t-else="" class="alert alert-warning">This product has no valid combination.</p>
-                            <div id="product_attributes_simple">
+                            <div
+                                t-if="not is_view_active('website_sale_comparison.accordion_specs_item')"
+                                id="product_attributes_simple"
+                            >
```

Comment:
Just noticed a weird behavior **not linked to my change**s, when a product is set with a single attribute, the attribute will be displayed regardless if `website_sale_comparison` is installed. (I created a new attribute with a single value and applied it on the warranty product for the sake of the example)
![image](https://github.com/user-attachments/assets/2fc348c1-9434-4c4a-8c81-0edf9d0e4091)

If the comparison module is installed the attribute will be displayed regarding the option:
- Bottom -> at the bottom of the page
- Accordion -> In the accordion
- None -> **It will still be displayed under the product description if it's a single value**

Is this really necessary ? Why do we differentiate the display of a single attribute, in this logic shouldn't all the attributes be displayed regardless if the module `website_sale_comparison` is installed. It feels weird to be in between for single attributes.

@AntoineVDV @dalaOdoo

Reply:
I guess the logic is that attributes with more than one value are displayed just above as choices for customer.
So they just wanted to show the specifications but I agree it's not clear. We should improve that in the future but not sure we want to do this now